server/assertions/selectors/selector_test.go

```diff
@@ -69,57 +69,67 @@ func TestSelector(t *testing.T) {
 		ExpectedSpanIds []trace.SpanID
 	}{
 		{
-			Name:            "Empty selector should select all spans",
+			Name:            "Empty_selector_should_select_all_spans",
```

Comment:
do you mind making this `CamelCase`? it's more consistent with go's naming

Reply:
we would have to change not just here, but most of our tests.

Reply:
yea, I've been doing that in the tests I updated